api/routes/assessment_routes.py

```diff
@@ -22,3 +25,28 @@ def all_assessments_for_fund_round_id(
     )
 
     return app_list
+
+
+def get_assessor_task_list_state(application_id: str) -> dict:
+    """get_assessor_task_list_state Function used by the endpoint
+    `/assessor_task_list/{application_id}`.
+
+    :param application_id: The stringified application UUID.
+    :return: A dictionary.
+    """
+
+    metadata = find_assessor_task_list_state(application_id)
+    sections, criterias = transform_to_assessor_task_list_metadata(
+        metadata["fund_id"], metadata["round_id"]
+    )
+
+    # We don't need to return this state.  It's only used to get the config above.
+    del metadata["round_id"]
```

Comment:
`del` is not typically used in python for filter since it's not very explicit and can create issues because dictionarys are typically passed by reference rather then value - i'd recommand writing a dictionary comprehension which filters and creates a new filtered dict to return.

Reply:
Thanks for noticing that, have refactored in https://github.com/communitiesuk/funding-service-design-assessment-store/pull/50/commits/cf75286e37b0cfe1f95423e8aea0ae793ec6502d